app/constants/saleable-options.js

```diff
@@ -0,0 +1,9 @@
+export const SALEABLE_OPTIONS = [
+  { id: "Not Set", translation_key: "items.new.not_set", value: null },
+  { id: "Saleable", translation_key: "items.new.saleable", value: "true" },
```

Comment:
how come the booleans are strings ?

Reply:
if i am going with boolean the null field is converted to false even if i want to set null.